pkg/trivyoperator/config.go

```diff
@@ -56,6 +56,7 @@ const (
 	keyConfigAuditReportsScanner         = "configAuditReports.scanner"
 	keyScanJobTolerations                = "scanJob.tolerations"
 	keyScanJobAnnotations                = "scanJob.annotations"
+	keyScanJobPodSecurityContext         = "scanJob.podSecurityContext"
```

Comment:
```suggestion
	keyScanJobPodSecurityContext         = "scanJob.podTemplateSecurityContext"
```

To be better aligned with the existing property `scanJob.podTemplateLabels`?